input/resources/au-core-actor-responder.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ActorDefinition xmlns="http://hl7.org/fhir">
+  <id value="au-core-actor-responder"/>
+  <url value="http://hl7.org.au/fhir/core/ActorDefinition/au-core-actor-responder" />
+  <name value="AUCoreResponder"/> 
+  <title value="AU Core Responder"/> 
+  <status value="draft"/>
+  <description value="The AU Core Responder is a system responsible for providing responses to queries submitted by AU Core Requesters."/> 
+  <type value="system"/> 
+  <documentation value="An AU Core Responder, also known as AU Core Server:&#xa;&#xa;
+- **SHALL** all *Must Support* elements if they know a value for the element&#xa;&#xa;
+- **SHALL** implement the requirements on [Suppressed Data](general-requirements.html#suppressed-data) when an element is NOT allowed to be shared&#xa;&#xa;
+- **SHALL** implement the requirements on [Missing data](general-requirements.html#missing-data) when an element value is NOT known:&#xa;&#xa;
```

Comment:
Change link text "Missing data" to "Missing Data" to make it consistent with the section title.

Reply:
fixed